### ISSUE TITLE
Shaka repos are moving

### ISSUE THREAD

**Issue by joeyparrish** [shaka-project/shaka-player#4007]
*TL;DR: Repos are moving to the shaka-project org, master branch will be renamed to main*

Today I'm announcing that the Shaka family of projects are migrating from [github.com/google](https://github.com/google) to the newly-created [github.com/shaka-project](https://github.com/shaka-project) organization.  This will let us more centrally manage repo settings, pool testing resources for GitHub Actions, and create a simple collection of our many projects.  (Probably [more than you realized we had](https://github.com/orgs/shaka-project/repositories).)

Some less-popular projects have already been moved.  The following projects will be moved soon (exact date uncertain):

|old|new|
|---|---|
|google/shaka-player|shaka-project/shaka-player|
|google/shaka-packager|shaka-project/shaka-packager|
|google/shaka-player-embedded|shaka-project/shaka-player-embedded|
|google/shaka-streamer|shaka-project/shaka-streamer|
|google/generic-webdriver-server|shaka-project/generic-webdriver-server|
|google/eme_logger|shaka-project/eme_logger|
|google/eme-encryption-scheme-polyfill|shaka-project/eme-encryption-scheme-polyfill|

You should not need to change anything for this.  GitHub will forward requests to the new locations automatically.

Also, to keep up with [evolving standard practices and inclusive language](https://github.com/github/renaming), the four Shaka projects' default branches will be renamed from master to main, right after the transfer.  You can update your local clones by running:

```sh
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
git remote prune origin
```

If you have any questions, please feel free to comment on GitHub or email me directly.

Thanks!
Joey, on behalf of the Shaka Team

**Comment by joeyparrish** [comment]
The move is now complete, and the branches have been renamed.  Thanks!
